Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:latest
+
+WORKDIR /var/adopteitor-api
+COPY package.json package.json
+COPY package-lock.json package-lock.json
+RUN npm install
+
+COPY . .
```

Comment:
This will be overridden by what's in `volumes`  in docker-compose file.

Our local dev files will be used, instead of these ones that got copied into the container.